{"name": "Find pets by incorrect status", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected error message: b'{\"code\":400,\"message\":\"Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`\"}' instead of Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`", "trace": "self = <tests_project.tests.test_find_by_status.TestFindByStatus object at 0x7f8321dcdba0>\n\n    @allure.severity(allure.severity_level.NORMAL)\n    @allure.title(\"Find pets by incorrect status\")\n    def test_find_pets_by_incorrect_status(self):\n        incorrect_status = \"free\"\n        response = MyRequests.get(\n            f\"/pet/findByStatus?status={incorrect_status}\")\n    \n        Assertions.assert_code_status(response, 400)\n>       Assertions.assert_error_message(\n            response, f\"Input error: query parameter `status value `{incorrect_status}` is not in the allowable values `[available, pending, sold]`\")\n\ntests/test_find_by_status.py:75: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [400]>\nexpected_error_message = 'Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`'\n\n    @staticmethod\n    @allure.step(\"Check the server returns error message '{expected_error_message}'\")\n    def assert_error_message(response: Response, expected_error_message):\n>       assert response.content.decode() == expected_error_message, \\\n            f\"Unexpected error message: {response.content} instead of {expected_error_message}\"\nE       AssertionError: Unexpected error message: b'{\"code\":400,\"message\":\"Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`\"}' instead of Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`\n\nlib/assertions.py:46: AssertionError"}, "steps": [{"name": "GET request to URL '/pet/findByStatus?status=free'", "status": "passed", "start": 1744662020870, "stop": 1744662020878}, {"name": "Check status code is 400", "status": "passed", "parameters": [{"name": "response", "value": "<Response [400]>"}, {"name": "expected_status_code", "value": "400"}], "start": 1744662020878, "stop": 1744662020878}, {"name": "Check the server returns error message ''Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`''", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected error message: b'{\"code\":400,\"message\":\"Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`\"}' instead of Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`\n", "trace": "  File \"/usr/local/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/tests_project/lib/assertions.py\", line 46, in assert_error_message\n    assert response.content.decode() == expected_error_message, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "response", "value": "<Response [400]>"}, {"name": "expected_error_message", "value": "'Input error: query parameter `status value `free` is not in the allowable values `[available, pending, sold]`'"}], "start": 1744662020879, "stop": 1744662020879}], "start": 1744662020870, "stop": 1744662020879, "uuid": "931832db-69ac-4182-b60a-5000d0967211", "historyId": "fbe0265c58a3833e7af6eea29943d36e", "testCaseId": "fbe0265c58a3833e7af6eea29943d36e", "fullName": "tests.test_find_by_status.TestFindByStatus#test_find_pets_by_incorrect_status", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_find_by_status"}, {"name": "subSuite", "value": "TestFindByStatus"}, {"name": "host", "value": "4c2631702ffc"}, {"name": "thread", "value": "7-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_find_by_status"}]}